apps/communications/dialer/js/dialer.js

```diff
@@ -36,9 +36,13 @@ var NavbarManager = {
 var CallHandler = (function callHandler() {
   var COMMS_APP_ORIGIN = document.location.protocol + '//' +
     document.location.host;
+  var callScreenWindow = null;
```

Comment:
I agree, I'll file a follow up for that but for now I didn't want to make this patch too different from a plain revert of the original to minimize risk.